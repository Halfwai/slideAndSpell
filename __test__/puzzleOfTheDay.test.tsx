import React from 'react';
import { render } from '@testing-library/react-native';
import PuzzleOfTheDay from '@/app/puzzleOfTheDay';

beforeEach(() => {
    jest.useFakeTimers();
});

afterEach(() => {
    jest.clearAllMocks();
    jest.useRealTimers();
});

jest.mock('expo-router', () => ({
    useRouter: jest.fn(() => ({
      push: jest.fn(),
    })),
  }));

it(`PuzzleOfTheDay Component renders correctly`, () => {
    const { toJSON } = render(
        <PuzzleOfTheDay/>
    );
    expect(toJSON()).toMatchSnapshot();
});