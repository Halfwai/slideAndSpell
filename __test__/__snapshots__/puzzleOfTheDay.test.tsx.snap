// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PuzzleOfTheDay Component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#CBE6F7",
      "flex": 1,
      "justifyContent": "center",
    }
  }
/>
`;
